src/main/java/housing/HousingMarket.java

```diff
@@ -7,6 +7,7 @@
 import org.apache.commons.math3.distribution.GeometricDistribution;
 import org.apache.commons.math3.random.MersenneTwister;
 
+import collectors.HousingMarketStats;
```

Comment:
As I mentioned before, it is because I used it previously, but it was not removed automatically in the Section of import when I deleted the code in the main body.

Reply:
Ok.